skyportal/handlers/api/internal/source_views.py

```diff
@@ -74,3 +78,8 @@ def post(self, obj_id):
         DBSession.add(sv)
         self.verify_and_commit()
         return self.success()
+
+
+def tIndex(t):
```

Comment:
```suggestion
def t_index(t):
```
? I'm surprised the linter didn't complain...